eda.Rmd

```diff
@@ -0,0 +1,213 @@
+
+clean_survey_all_days
+typeof(clean_survey_all_days$distance_km)
+
+dim(survey_data)
+dim(clean_survey_sep_days)
+dim(clean_survey_all_days)
+
+```
+
+### TABLE OF SUMMARY STATISTICS
+```{r}
+summary(clean_survey_all_days) %>% 
+  knitr::kable()
+
+summary(clean_survey_sep_days) %>% 
+  knitr::kable()
+
+
+# by mode of transit
+by(clean_survey_all_days, clean_survey_all_days$mode_of_transport, summary)
+```
+
+### EDA
+
+```{r}
+### INITIAL HISTOGRAMS FOR MW & TT
+
+clean_survey_sep_days %>% 
+  ggplot(aes(x = mw_arrival)) +
+  geom_histogram()
+
+clean_survey_sep_days %>% 
+  ggplot(aes(x = tt_arrival)) +
+  geom_histogram()+
+  labs(x="Arrival time", y="Frequency", title = "Arrival time v/s number of students")
+
+```
+
+
+```{r}
+### INITIAL SCATTERPLOTS FOR MW
+clean_survey_sep_days %>% 
+  ggplot(aes(x = mw_arrival, y = distance_km)) +
+  geom_point()+
+  labs(x="Arrival time", y="Distance(in kms)", title = "Arrival time v/s distance on Mondays and Wednesdays")
+```
+
+```{r}
+### FACET ON MODE OF TRANSPORT
+clean_survey_sep_days %>% 
+  ggplot(aes(x = mw_arrival, y = distance_km)) +
+  geom_point() +
+  facet_wrap(~ mode_of_transport)+
+  labs(x="Arrival time", y="Distance(in kms)", title = "Arrival time v/s distance")
+
+clean_survey_sep_days %>% 
+  ggplot(aes(x = mw_arrival, y = distance_km)) +
+  geom_point() +
+  facet_wrap(~ mode_of_transport) +
+  geom_smooth(method = "lm", se = F)+
+  labs(x="Arrival time", y="Distance(in kms)", title = "Arrival time v/s distance")
+```
+
+```{r}
+### COLOR INSTEAD OF FACET
+clean_survey_sep_days %>% 
+  ggplot(aes(x = mw_arrival, y = distance_km)) +
+  geom_point(aes(color = mode_of_transport))+
+  labs(x="Arrival time", y="Distance(in kms)", title = "Arrival time v/s distance")
+```
+
+
+```{r}
+### SCATTERPLOT FOR ALL DAYS, AND FACETED
+
+clean_survey_all_days %>% 
+  ggplot(aes(x = arrival, y = distance_km)) +
+  geom_point()+
+  labs(x="Arrival time", y="Distance(in kms)", title = "Arrival time v/s distance")
+
+clean_survey_all_days %>% 
+  ggplot(aes(x = arrival, y = distance_km)) +
+  geom_point(aes(color = mode_of_transport), alpha = .5) +
+  facet_wrap(~ mode_of_transport)+
+  labs(x="Arrival time", y="Distance(in kms)", title = "Arrival time v/s distance")
+```
+
+
+```{r}
+### DENSITY 
+clean_survey_all_days %>% 
+  ggplot(aes(x = arrival)) +
+  geom_density() + 
+  facet_wrap(~ mode_of_transport)+
+  labs(x="Arrival time", y="Frequency", title = "Count of student arrival time")
+```
+
+
+```{r}
+### BOXPLOT & VIOLIN PLOT FOR ALL DAYS
+
+clean_survey_all_days %>% 
+  ggplot(aes(x = mode_of_transport, y = arrival)) +
+  geom_boxplot()+
+  labs(y="Arrival time", x="Mode of transport", title = "Mode of transport v/s arrival time")
+
+
+clean_survey_all_days %>% 
+  ggplot(aes(x = mode_of_transport, y = arrival)) +
+  geom_violin() +
+  geom_jitter(width = .2, alpha = .3)+
+  labs(y="Arrival time", x="Mode of transport", title = "Mode of transport v/s arrival time")
+```
+
+### Conclusion
```

Comment:
since we haven't actually run our analysis yet I don't think we should draw any conclusions yet. Maybe re-phrase this?

Reply:
Conclusion based EDA (agreed)